modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/RichTextField.testcase

```diff
@@ -11,6 +11,18 @@ definition {
 		User.firstLoginPG();
 
 		JSONGroup.addGroup(groupName = "Test Site Name");
+
+		var structureName = "Test LPS-98076";
```

Comment:
Hey @nicolas-sss , could you please don't use the LPS tickets as values for variables?

Thanks!